.github/workflows/create-gui-pr.yml

```diff
@@ -1,4 +1,4 @@
-name: create-gui-pr
+name: Update kuma with latest kuma-gui
```

Comment:
I would rather not change this name. It’s the one that shows up in overviews like https://github.com/kumahq/kuma-gui/actions:

![image](https://user-images.githubusercontent.com/5774638/188078333-4a3b57af-d446-4b24-8214-00d09e97a0e1.png)

... and PR status checks like here:

![image](https://user-images.githubusercontent.com/5774638/188078498-49bfb7e4-8bec-4e3c-9fce-2d9e423cc485.png)

Reply:
Shouldn't they have a readable name?

Reply:
Or do you mean there's a problem with changing the name in general?

Reply:
There is no technical problem with changing the name, no. They can be called like you suggested. I just think it’s quite noisy for the status checks, for example, to read “Update kuma with latest kuma-gui / Create PR to update kuma-gui”.

In any case, it’s a preference and a nitpick. I don’t mind to change them, but I personally wouldn’t.